roles/k8s/files/monitoring/blackbox-exporter.yaml

```diff
@@ -30,26 +31,31 @@ spec:
         configMap:
           name: blackbox-exporter-config
 ---
+# require service for scraping
+apiVersion: v1
+kind: Service
+metadata:
+  name: blackbox-exporter
+  namespace: monitoring
+spec:
+  selector:
+    app: blackbox-exporter
+  ports:
+  - protocol: TCP
+    port: 9115
+    targetPort: 9115
```

Comment:
you need service to use blackbox exporter
(blackbox exporter requires different config compared to other exporter)